src/WooCommerce.php

```diff
@@ -575,14 +575,16 @@ public static function woocommerce_email_order_meta($order) {
     if ($subscriber_id = get_user_meta($user_id, 'subscriber_id', TRUE)) {
       echo '<p><strong>' . __('Subscription ID:', PLUGIN::L10N) . '</strong> ' . $subscriber_id . '</p>';
     }
-    $userinfo = get_user_meta($user_id, Plugin::USER_META_USERINFO, TRUE);
-    if (empty($userinfo['id'])) {
-      throw new \LogicException('Unable to build user info: Missing SSO ID.');
+    if (!isset($_POST['optins'])) {
```

Comment:
`$_POST['optins']` does not exist; how did this work for you…?

Reply:
> At the time of a checkout/purchase, there is no userinfo right now. So for the initial order email, you can only consult the POST data of the order, but not the userinfo.

Obviously I am misunderstanding this.